lib/form_data.js

```diff
@@ -177,9 +177,28 @@ FormData.prototype._multiPartHeader = function(field, value, options) {
     'Content-Type': [].concat(contentType || [])
   };
 
+  // allow custom headers.
+  if (typeof options.header == 'object') {
+    populate(headers, options.header);
+  }
+
+  var header;
   for (var prop in headers) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [The body of a for in should be wrapped in an if statement to filter unwanted properties from the prototype.](https://www.codacy.com/app/form-data/form-data/file/2355437101/issues/source?bid=2805153&fileBranchId=3218876#l186)